class/openshift-prometheus-proxy.yml

```diff
@@ -1,5 +1,9 @@
 parameters:
   kapitan:
+    dependencies:
+      - type: https
+        source: https://raw.githubusercontent.com/appuio/openshift-prometheus-proxy/master/template/openshift-prometheus-proxy.yaml
```

Comment:
is referencing `master` a good idea here?

Reply:
Maybe not, I haven't thought about that in too much detail yet. I'll change it to use a version from the hierarchy